src/Evolution/VariableFixing/LimitLorentzFactor.hpp

```diff
@@ -0,0 +1,97 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+// IWYU pragma: no_forward_declare hydro::Tags::SpatialVelocity
+// IWYU pragma: no_forward_declare hydro::Tags::LorentzFactor
+// IWYU pragma: no_forward_declare hydro::Tags::RestMassDensity
+// IWYU pragma: no_forward_declare Tensor
+
+namespace VariableFixing {
+/*!
+ *\ingroup VariableFixingGroup
+ * \brief Limit the maximum Lorentz factor to LorentzFactorCap in regions where
+ * the density is below MaxDensityCutoff.
+ *
+ * The Lorentz factor is set to LorentzFactorCap and the spatial velocity is
+ * adjusted according to:
+ * blah
```

Comment:
...

Reply:
derp :( doxygen wasn't rendering things correctly and I was debugging it. Cleaned up now :)